pkg/kademlia/routing_integration_test.go

```diff
@@ -0,0 +1,616 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package kademlia
```

Comment:
Le'ts mark tests as `kademila_test`, such tests are less likely to fail

Reply:
this and moving dht.RoutingTable to a test file seem to create an import cycle. Any advice for that?